src/imap.rs

```diff
@@ -18,6 +19,7 @@ use crate::param::Params;
 use crate::wrapmime;
 
 const DC_IMAP_SEEN: usize = 0x0001;
+const DCC_IMAP_DEBUG: &str = "DCC_IMAP_DEBUG";
```

Comment:
not sure where to put it so that people would see it.  Those who are into debugging, or where we are trying to understand a connection issue, we can tell "set DCC_IMAP_DEBUG" in IRC etc.